### ISSUE TITLE
Review grey labels

### ISSUE THREAD

**Issue by gusliedke** [comicrelief/component-library#337]
Input labels are failing colour contrast.

Add new grey or change all labels.

**Comment by gusliedke** [comment]
@Ruby-HB These are the current greys.
```
  grey: '#969598',
  grey_dark: '#222222',
  grey_extra_light: '#f0f0f0',
  grey_for_forms: '#666',
  grey_light: '#F4F3F5',
  grey_medium: '#E1E2E3',
```
let me know if you need a new one or which we use for the labels.

**Comment by Ruby-HB** [comment]
Please try this grey - it would be 'grey_deep'
(Also this will probably be used in forms too so will come in handy!)
Thanks :)

**Comment by gusliedke** [comment]
![image](https://user-images.githubusercontent.com/1434022/90149406-5cc02180-dd7c-11ea-9ef6-e46bfa1eee5d.png)

![image](https://user-images.githubusercontent.com/1434022/90149637-9f81f980-dd7c-11ea-910a-48c922045c19.png)
``` #616161 ``` works
